tools/proc_globalids.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+import math
+import numpy as np
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+import operator
+import itertools
+
+from timeit import default_timer as timer
+from datetime import timedelta
+from msg_alltoall import get_global_node_ids 
+
+def getGlobalIdsForEdges( rank, size, edge_data, metis_partitions, node_data ): 
+
```

Comment:
remove the empty line and remove the empty space before and after the arguments.